lib/ansible/modules/cloud/amazon/rds_tag.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import (HAS_BOTO3, boto3_conn, ec2_argument_spec, get_aws_connection_info)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: rds_tag
+short_description: create and remove tag(s) to rds instance.
+description:
+    - Creates, removes and lists tags from any rds instance. The instance is referenced by its instance name.
+      It is designed to be used with complex args (tags), see the examples.  This module has a dependency on python-boto3.
+options:
+  instance_name:
+    description:
+      - The RDS instance name.
+  state:
+    description:
+      - Whether the tags should be present or absent on the resource. Use list to interrogate the tags of an instance.
+    default: present
+    choices: ['present', 'absent', 'list']
+  tags:
+    description:
+      - a hash/dictionary of tags to add to the resource; '{"key":"value"}' and '{"key":"value","key":"value"}'
+    required: true
+'''
+
+EXAMPLES = '''
+- name: Ensure tags are present on a resource
+  rds_tag:
+    region: eu-west-1
+    instance_name: name-database
+    state: present
+    tags:
+      Name: environment
+      env: sandbox
+'''
+
+try:
```

Comment:
This would be handled by the `AnsibleAWSModule`

```python
try:
    import botocore
except ImportError:
    pass  # handled by AnsibleAWSModule
```